GoogleForCreators/web-stories-wp
# Reorganize video settings in design panel

## Summary

Re-organize the video settings in design panel by follow the structure:
**Playback**
    Loop / play once
**Poster**
    [Poster selector]
    "For improved loading experience on slower connections."
**Description**
    [Text field]
    "For indexability and accessibility. Include any burned-in text inside the video."
**Captions**
    [Captions selector]

## Relevant Technical Choices
1. Update the Video Options panel title to "Playback"
2. Add a Poster element
3. Update the VideoAccessibility panel title to "Description"
4. Clean up the captions panel

## To-do

N/A

## User-facing changes

Update designing panel

![Screen Shot 2020-12-10 at 11 04 34 PM](https://user-images.githubusercontent.com/11252886/101935584-2a8cc800-3b94-11eb-94f1-5d2946dd8871.png)

## Testing Instructions

- Playback button should be switchable
- Should be able to upload poster
- Should be able to update description text
- Should be able to update captions

---

Fixes #5340
